.lint

```diff
@@ -1,3 +1,7 @@
 exclusions:
  template-job-rule:
  panel-job-instance-rule:
+ template-datasource-rule:
+  reason: CI failing because label has different caps, this was changed in grafana-builder
```

Comment:
Recently grafana-builder changed the label to match some internal linter, now the linter here fails, excluding this for now but can probably be enabled again when all linters agree on this.

Reply:
Attempting to fix this exclusion in https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/891